components/homme/src/preqx/share/prim_advance_mod.F90

```diff
@@ -825,9 +756,13 @@ subroutine advance_hypervis_dp(elem,hvcoord,hybrid,deriv,nt,nets,nete,dt2,eta_av
                  ! normalize so as to conserve IE
                  ! scale by 1/rho (normalized to be O(1))
                  ! dp/dn = O(ps0)*O(delta_eta) = O(ps0)/O(nlev)
+#if 0
```

Comment:
Does this mean this chunk of code is slated to be removed in the future?  If so, why are we keeping it?

Reply:
removed